### ISSUE TITLE
environment variable to save configured settings in a file

### ISSUE THREAD

**Issue by lunarpapillo** [KhronosGroup/Vulkan-ValidationLayers#7712]
Some application vendors configure the validation layer programmatically (i.e. via switches like `-vulkan_enable_validation -vulkan_enable_sync_validation` etc.).

For local analysis, it would be very helpful to know exactly how the validation layer is actually being configured in these cases.

Defining an environment variable `VK_VALIDATION_SAVE_SETTINGS=/path/to/file/to/save/settings` that caused VVL to write a settings file to the given location would make it easy.

**Comment by spencer-lunarg** [comment]
agree, this is also nice in general as sometimes I myself am concern I do/don't have certain settings enabled... `VK_LAYER_VALIDATION_SAVE_SETTINGS` would be a good debug env variable

**Comment by charles-lunarg** [comment]
Would this be something to add to the settings library instead of being specific to validation?

**Comment by spencer-lunarg** [comment]
@christophe-lunarg this is something we really want for other layers too, is this something Layer Settings can take care of?

**Comment by christophe-lunarg** [comment]
This looks great to me and yes the layer settings library is a good place for this to generalize to all layers.
We could also have a `VK_VALIDATION_LOAD_SETTINGS` env variable.

Probably a bit tricky to handle correcly the interaction of multiple layers on that file. I think we should go for a single environment variable `VK_LAYERS_LOAD_SETTINGS` and `VK_LAYERS_SAVE_SETTINGS` shared across layers.

Does it sounds good to you?

Should we move this issue in the VulkanUtility repository?
